src/index.js

```diff
@@ -25,19 +27,35 @@ async function runTest(files, reporter, requireFilePath) {
             allTestFiles.push(requireFilePath);
         }
         allTestFiles.push(...files);
-        await brs.execute(allTestFiles, {
+        let returnVals = await brs.execute(allTestFiles, {
             stdout: reporterStream,
             stderr: process.stderr
         });
+
         reporterStream.end();
+
+        if (forbidFocused && returnVals.length > 0) {
+            // iterate through return values and see if there are focused cases
+            for (output of returnVals) {
+                let focusedFiles = output.getValue().get("fileswithfocusedcases").getElements();
+                if (focusedFiles.length > 0) {
+                    let formattedList = focusedFiles.map((brsStringPath) => `    ${brsStringPath.value}`).join("/n");
```

Comment:
I think you've got the wrong slash for this escape sequence 🤓 
```suggestion
                    let formattedList = focusedFiles.map((brsStringPath) => `    ${brsStringPath.value}`).join("\n");
```